src/App.tsx

```diff
@@ -1,93 +1,59 @@
-import React, { useEffect, useState } from 'react';
-
+import React, { useState, useEffect } from 'react';
 import './App.scss';
 import './styles/general.scss';
-
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
-import { getAllUsers } from './api/users';
-import { getAllPosts, getUserPosts } from './api/posts';
-
-import { Loader } from './components/Loader';
+import { getAllUsers } from './api/api';
+import { User } from './react-app-env';
 
 const App: React.FC = () => {
-  const [users, setUsers] = useState<User[]>([]);
-  const [posts, setPosts] = useState<Post[]>([]);
-  const [isLoading, setIsLoading] = useState(true);
-  const [postId, setPostId] = useState(0);
-  const [selectedUserId, setSelectedUserId] = useState(0);
-
-  const fetchUsers = async () => {
-    setIsLoading(true);
-
-    const usersFromServer = await getAllUsers();
-
-    setUsers(usersFromServer);
-  };
-
-  const fetchPosts = async () => {
-    const postsFromServer = await getAllPosts();
-
-    setPosts(postsFromServer);
-    setIsLoading(false);
-  };
-
-  const fetchUserPosts = async () => {
-    setIsLoading(true);
-    const postsUsersFromServer = await getUserPosts(selectedUserId);
-
-    setPosts(postsUsersFromServer);
-    setIsLoading(false);
-  };
+  const [allUsers, setAllUsers] = useState<User[]>([]);
+  const [currentUser, setCurrentUser] = useState('0');
```

Comment:
The line "useState('0')" can be improved by using "useState<string>('0')" instead. This is because it explicitly declares the type of the state variable as a string, which can prevent potential errors and improve code readability. By using this syntax, TypeScript will also be able to provide more accurate type-checking and better code suggestions.